gitlab-ldap-sync.py

```diff
@@ -24,7 +33,11 @@
         if config['log_level']:
             log_option['level'] = getattr(logging, str(config['log_level']).upper())
         logging.basicConfig(**log_option)
-        print('Done.')
+        if config['cron'] and config['log']:
+            console = logging.StreamHandler()
+            console.setLevel(logging.ERROR)
+            logging.getLogger('').addHandler(console)
```

Comment:
I don't think we need this part. We already suppress "print" logs and you can set the log level in `config.json`